XenAdmin/Diagnostics/Checks/SafeToUpgradeCheck.cs

```diff
@@ -76,8 +77,8 @@ protected override Problem RunCheck()
         }
 
         public override string Description
-        {
-            get { return string.Empty; }
+        {
+            get { return Messages.CHECKING_SAFE_TO_UPGRADE_HOST; }
```

Comment:
Better to be "SAFE_TO_UPGRADE_HOST_DESCRIPTION", consistent with other parallel macros.